settings.gradle

```diff
@@ -1,5 +1,3 @@
-import org.apache.tools.ant.taskdefs.condition.Os
-
```

Comment:
not sure, but you may still need this

Reply:
it was not needed previously. local build is fine without it.